.github/workflows/ci.yml

```diff
@@ -8,16 +8,20 @@ jobs:
       matrix:
         node-version: [8.x, 10.x, 12.x]
     steps:
-    - uses: actions/checkout@v1
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
-      with:
-        node-version: ${{ matrix.node-version }}
-    - name: Install dependencies
-      run: yarn install --frozen-lockfile
-    - name: Run linter
-      run: yarn lint
-    - name: Run tests
-      run: yarn test
-    env:
-      CI: true
+      - uses: actions/checkout@v1
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
+      - name: Run linter
+        run: yarn lint
+      - name: Run tests
+        run: yarn test:coverage
+        env:
```

Comment:
I wonder if we should move this to be outside of this particular step so the entire job runs with `CI: true`?

Reply:
Sure! I wasn't sure where that was supposed to be